pkg/get.go

```diff
@@ -134,3 +135,15 @@ func GetUID(engineName, namespace string, clients environment.ClientSets) (strin
 	}
 	return string(chaosEngine.UID), nil
 }
+
+func GetRunHistoryStatus(testsDetails *types.TestDetails, clients environment.ClientSets) v1alpha1.HistoryDetails {
```

Comment:
can we give comment on the function - what is does? -- check at other places as well.